contracts/src/bridge/Inbox.sol

```diff
@@ -233,31 +225,21 @@ contract Inbox is DelegateCallAware, PausableUpgradeable, IInbox {
             // have the L1 sender address mapped.
             // Here we preemptively reverse the mapping for EOAs so deposits work as expected
             sender = AddressAliasHelper.undoL1ToL2Alias(sender);
-        } else {
-            destinationAddress = AddressAliasHelper.applyL1ToL2Alias(destinationAddress);
         }
 
         return
             _deliverMessage(
-                L1MessageType_submitRetryableTx,
-                sender,
-                abi.encodePacked(
-                    // the beneficiary and other refund addresses don't get rewritten by arb-os
-                    // so we use the original msg.sender value
-                    uint256(uint160(bytes20(destinationAddress))),
-                    uint256(0),
-                    msg.value,
-                    maxSubmissionCost,
-                    uint256(uint160(bytes20(destinationAddress))),
-                    uint256(uint160(bytes20(destinationAddress))),
-                    uint256(0),
-                    uint256(0),
-                    uint256(0),
-                    ""
-                )
+                L1MessageType_ethDeposit,
+                sender, // arb-os will add the alias to this value
+                abi.encodePacked(msg.value)
             );
     }
 
+    /// @notice deprecated in favour of depositEth with no parameters
+    function depositEth(uint256) external payable virtual override whenNotPaused returns (uint256) {
+        return depositEth();
```

Comment:
Is there concern about contracts which require L2 execution when they deposit ETH? That was theoretically possible before right?

Reply:
even though its theoretically possible, my thinking is that if they want L2 execution they should be using retryable tickets instead of this entrypoint